src/management_api_service.ts

```diff
@@ -527,6 +531,15 @@ export class ManagementApiService implements IManagementApi {
     return this.kpiApiService.getActiveTokensForFlowNode(identity, flowNodeId);
   }
 
+  public async getFlowNodeInstancesForProcessInstance(
+    identity: IIdentity,
+    processInstanceId: string,
+  ): Promise<Array<DataModels.FlowNodes.FlowNodeInstance>> {
```

Comment:
With regards to my remarks [here](https://github.com/process-engine/management_api_contracts/pull/50)

```suggestion
  ): Promise<Array<DataModels.FlowNodeInstances.FlowNodeInstance>> {
```